diff --git a/node_modules/microbundle/dist/cli.js b/node_modules/microbundle/dist/cli.js
index 82ca83e..5201384 100755
--- a/node_modules/microbundle/dist/cli.js
+++ b/node_modules/microbundle/dist/cli.js
@@ -1049,7 +1049,7 @@ function createConfig(options, entry, format, writeMeta) {
       }), customBabel()({
         babelHelpers: 'bundled',
         extensions: EXTENSIONS,
-        exclude: 'node_modules/**',
+        exclude: /\/node_modules\//,
         passPerPreset: true,
         // @see https://babeljs.io/docs/en/options#passperpreset
         custom: {
